pymc_marketing/clv/models/pareto_nbd.py

```diff
@@ -632,16 +646,17 @@ def _distribution_new_customers(
             s = pm.HalfFlat("s")
             beta = pm.HalfFlat("beta")
 
-            # This is the shape if using fit_method="map"
-            if self.fit_result.dims == {"chain": 1, "draw": 1}:
-                shape_kwargs = {"shape": 1000}
-            else:
-                shape_kwargs = {}
+            pm.Gamma("population_purchase_rate", alpha=r, beta=1 / alpha, shape=1000)
+            pm.Gamma("population_dropout", alpha=s, beta=1 / beta, shape=1000)
```

Comment:
This doesn't make sense for a full mcmc fit? Why do you need a 4k * 1k samples?

Reply:
I could use some help to fix this:  the mock idata object for a`mcmc` fit isn't triggering the IF statement in testing, so it has the same dims as a MAP fit. It seems to work fine when tested in the notebook though.

Also, `distribution_customer_population` has an additional dimension due to the `T` parameter. I've added an argument to make this adjustable, but it may be more trouble than it's worth.